cli/flags.rs

```diff
@@ -793,32 +797,6 @@ pub enum DenoSubcommand {
   Version,
 }
 
-fn get_default_bundle_filename(source_file: &str) -> String {
-  let specifier = ModuleSpecifier::resolve_url_or_path(source_file).unwrap();
-  let path_segments = specifier.as_url().path_segments().unwrap();
-  let file_name = path_segments.filter(|s| !s.is_empty()).last().unwrap();
-  let file_stem = file_name.trim_end_matches(".ts").trim_end_matches(".js");
-  format!("{}.bundle.js", file_stem)
-}
-
-#[test]
-fn test_get_default_bundle_filename() {
-  assert_eq!(get_default_bundle_filename("blah.ts"), "blah.bundle.js");
-  assert_eq!(
-    get_default_bundle_filename("http://example.com/blah.ts"),
-    "blah.bundle.js"
-  );
-  assert_eq!(get_default_bundle_filename("blah.js"), "blah.bundle.js");
-  assert_eq!(
-    get_default_bundle_filename("http://example.com/blah.js"),
-    "blah.bundle.js"
-  );
-  assert_eq!(
-    get_default_bundle_filename("http://zombo.com/stuff/"),
-    "stuff.bundle.js"
-  );
-}
```

Comment:
Note this was removed because of

> Change of behaviour when an output file is not specified on the CLI. Previously a default name was determined and the bundle written to that file, now the bundle will be sent to stdout.

👍